custom-metrics-stackdriver-adapter/pkg/adapter/provider/provider.go

```diff
@@ -44,6 +45,8 @@ import (
 const (
 	nodeResource = "nodes"
 	podResource  = "pods"
+	// Maximum number of entries in the external metrics LRU cache.
+	defaultExternalMetricsCacheSize = 1024
```

Comment:
@raywainman @CatherineF-dev do you think we need to expose this also as a flag to be configurable by users?

Reply:
Removed the default values and using whatever values are passed to flags for now. This is because, we want to provide the cache as an optional configuration - without any defaults. Can revisit this later once we enable the cache by default.